src/dppm_rest_api.cr

```diff
@@ -28,12 +28,14 @@ module DppmRestApi
     host : String,
     port : Int32,
     data_dir : String,
+    prefix : String = DppmRestApi.default_prefix,
     access_filter : Proc(HTTP::Server::Context, Access, Bool) = ->access_filter(HTTP::Server::Context, Access)
   )
     ::File.open Path[data_dir, PERMISSIONS_FILE] do |data|
       @@permissions_config = Config.from_json data
     end
 
+    DppmRestApi.permissions_config.prefix = DPPM::Prefix.new(prefix).tap &.create
```

Comment:
```suggestion
    DppmRestApi.permissions_config.prefix = DPPM::Prefix.new prefix
```
Bad idea, I don't think we should not create prefixes automagically.
The user have to explicitly do `dppm app install`. This can create a system service. This will be a way to start the API.
To check if the Prefix exists, use `Prefix#check`